test/distributed/_tensor/debug/test_op_coverage.py

```diff
@@ -0,0 +1,34 @@
+# Owner(s): ["oncall: distributed"]
+
+import torch
+import torch.nn as nn
+
+from torch.distributed._tensor.debug.op_coverage import get_inductor_decomp_graphs
+
+from torch.testing._internal.common_utils import run_tests, TestCase
+
+
+class SimpleMLP(nn.Module):
+    def __init__(self):
+        super(SimpleMLP, self).__init__()
+        self.net1 = nn.Linear(50, 32)
+        self.relu = nn.ReLU()
+        self.net2 = nn.Linear(32, 8)
+
+    def forward(self, x):
+        return torch.sigmoid(self.net2(self.relu(self.net1(x))))
+
+
+class TestOpCoverage(TestCase):
+    def test_trace_with_inductor_decomp(self):
+        model = SimpleMLP()
+        args = (torch.randn(8, 50),)
+        kwargs = {}
+        graphs = get_inductor_decomp_graphs(model, args, kwargs)
+        assert len(graphs) == 2, "Expect fwd + bwd graphs"
+        self.assertIsInstance(graphs[0], torch.fx.GraphModule)
```

Comment:
question: will we check the content of the graphs in this test or another? Does this PR include checking the graph content or just validating that the util can return 1 fwd and 1 bwd graphs.

Reply:
This PR would not check the content of the graph in the tests, I added a `print_op_summary` function that would print the op coverage summary